src/video-grid/useMediaStream.ts

```diff
@@ -187,15 +195,17 @@ export const useSpatialMediaStream = (
   audioOutputDevice: string,
   audioContext: AudioContext,
   audioDestination: AudioNode,
-  mute = false
+  mute = false,
+  localVolume: number
 ): [RefObject<Element>, RefObject<MediaElement>] => {
   const tileRef = useRef<Element>();
   const [spatialAudio] = useSpatialAudio();
```

Comment:
If spatial audio is enabled, the volume control won't have any effect since audio gets routed separately from the media element via the Web Audio API. You'd need to add a [GainNode](https://developer.mozilla.org/en-US/docs/Web/API/GainNode) into the processing chain below